README.md

```diff
@@ -409,25 +502,30 @@ Skyflow Elements can be used to securely reveal data in an application without e
 ### Step 1: Create a container
 To start, create a container using the `skyflowClient.container(Skyflow.ContainerType.REVEAL)` method as shown below.
 ```kt
-val container = skyflowClient.container(type: Skyflow.ContainerType.REVEAL)
+val container = skyflowClient.container(type = Skyflow.ContainerType.REVEAL)
 ```
 
 ### Step 2: Create a reveal Element
 Then define a Skyflow Element to reveal data as shown below.
 ```kt
 val revealElementInput = Skyflow.RevealElementInput(
-        id: "string",
-        redaction: Skyflow.RedactionType.DEFAULT,
-        styles: Skyflow.Styles,        //optional, styles to be applied to the element
-        label: "cardNumber"            //optional, label for the element
+        token = "string",
+        redaction = Skyflow.RedactionType.DEFAULT,
+        inputStyles = Skyflow.Styles(),        //optional, styles to be applied to the element
+        labelStyles = Skyflow.Styles(),  //optional, styles to be applied to the label of the reveal element
+        errorTextStyles = Skyflow.Styles(),  //optional styles that will be applied to the errorText of the reveal element
+        label = "cardNumber"            //optional, label for the element,
+        altText = "XXXX XXXX XXXX XXXX" //optional, string that is shown before reveal, will show token if altText is not provided
     )
 ```
-The `styles` parameter accepts a styles object as described in the [previous section](#step-2-create-a-collect-element) for collecting data but the only state available for a reveal element is the base state. For a list of acceptable redaction types, see the [section above](#Retrieving-data-from-the-vault).
+The `inputStyles` parameter accepts a styles object as described in the [previous section](#step-2-create-a-collect-element) for collecting data but the only state available for a reveal element is the base state. For a list of acceptable redaction types, see the [section above](#Retrieving-data-from-the-vault).
```

Comment:
can you add the variants available for input styles, error styles and label styles with sample code snippets

Reply:
added in the create a collect element section, also mentioned here that only the base variant is applied.